loopchain/peermanager/peer_manager.py

```diff
@@ -59,16 +47,15 @@ def serialize_as_preps(self) -> list:
                 for peer_id, peer in self._peer_list_data.peer_list.items()]
 
     def load_peers(self) -> None:
-        PeerLoader.load(peer_manager=self)
         blockchain = ObjectManager().channel_service.block_manager.blockchain
-
-        reps_hash = self.reps_hash()
-        reps_in_db = blockchain.find_preps_by_roothash(reps_hash)
-
-        if not reps_in_db:
-            preps = self.serialize_as_preps()
-            util.logger.spam(f"in _load_peers serialize_as_preps({preps})")
-            blockchain.write_preps(reps_hash, preps)
+        if not blockchain.roothash_exist_in_db(self._crep_root_hash):
+            PeerLoader.load(peer_manager=self)
+            reps_hash = self.reps_hash()
+            reps_in_db = blockchain.roothash_exist_in_db(reps_hash)
```

Comment:
Expected different return value caused by this line. `roothash_exist_in_db` returns boolean.

https://github.com/icon-project/loopchain/pull/459/files#r364537455